signac/contrib/project.py

```diff
@@ -2625,17 +2619,19 @@ def __iter__(self):
             self._project, self._project._find_job_ids(self._filter)
         )
 
+    @deprecated(
+        deprecated_in="1.8",
```

Comment:
I wasn't sure what number to put there since the old warning has apparently been there forever and nobody saw it (including us). But I agree, we can use the old version number even though clearly nobody saw that deprecation as long as we wait to remove until 2.0.

Reply:
This is correct. I'll apply @csadorf's suggestion.

We did not remove `next()` in 1.0.0 because it had been deprecated very recently. Source: https://github.com/glotzerlab/signac/pull/139